### PR TITLE
Contact Form using emailJS working

### PR DESCRIPTION
**Pull request by Pablo-Mdz**
---
name: Pull Request Template
about: Email form
---

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no new functionality, just code cleanup/refactoring)
- [ ] Documentation update

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)

### CODE REVIEW

File: client/src/pages/ContactMe/ContactMe.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+        <div className="mb-4">
+          <label
+            htmlFor="email"
+            className="block text-gray-700 font-medium mb-2"
+          >
+            {/* Email */}
+          </label>
+          <input
+            id="mailid"
```
Comment (antoniosaleme) [inline client/src/pages/ContactMe/ContactMe.jsx]: id and htmlFor should have the same value

File: client/src/pages/ContactMe/ContactMe.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+        <div className="mb-4">
+          <label
+            htmlFor="message"
+            className="block text-gray-700 font-medium mb-2"
+          >
+            {/* Message */}
+          </label>
+          <textarea
+            id="msg"
```
Comment (antoniosaleme) [inline client/src/pages/ContactMe/ContactMe.jsx]: same here

File: client/src/App.jsx [hunk 3]
```diff
@@ -29,6 +30,7 @@ function App() {
         <Route path="/topics/:id" element={<TopicDetails />} />
         <Route path="/create-post" element={<CreateAPost />} />
         <Route path="/create-topic" element={<CreateATopic />} />
+        <Route path='/contactMe' element={<ContactMe />} />
```
Comment (antoniosaleme) [inline client/src/App.jsx]: I think the correct way should be: "/contact-me"

File: client/src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -41,7 +41,7 @@ const Navbar = () => {
                                 Dictionary
                             </Link>
                             <Link
-                                to="/contact"
+                                to="/contactMe"
```
Comment (antoniosaleme) [inline client/src/components/Navbar/Navbar.jsx]: I think should be to="/contact-me"

**Review by antoniosaleme — CHANGES_REQUESTED** [review]

File: client/src/pages/ContactMe/ContactMe.jsx (hunk 1 above)
Comment (Pablo-Mdz) [reply client/src/pages/ContactMe/ContactMe.jsx]: i'll change it now

**Review by antoniosaleme — APPROVED** [review]
hermoso! ❤️ 🧟‍♂️

**Pull request merged**